examples/jpeg_es6/package.json

```diff
@@ -0,0 +1,19 @@
+{
+	"name": "jpeg-fuzz",
```

Comment:
This module has the same name as `jpeg` which leads to warnings during startup of the tests.
```text
jest-haste-map: Haste module naming collision: jpeg-fuzz
  The following files share their name; please adjust your hasteImpl:
    * <rootDir>/examples/jpeg/package.json
    * <rootDir>/examples/jpeg_es6/package.json
```

Reply:
Corrected